src/Features/CSharp/Portable/Completion/CompletionProviders/SnippetCompletionProvider.cs

```diff
@@ -157,7 +157,7 @@ SyntaxKind.EndRegionKeyword or
                 return ImmutableArray<CompletionItem>.Empty;
 
             var snippets = service.GetSnippetsIfAvailable();
-            if (context.CompletionOptions.ShouldShowNewSnippetExperience())
+            if (context.CompletionOptions.ShouldShowNewSnippetExperience() && !context.Document.IsRazorDocument())
```

Comment:
Should this be rolled into ShouldShowNewSnippetExperience?

Reply:
Could you pass in the `Document` as an argument to `ShouldShowNewSnippetExperience`?